src/ui/pages/equipment/components/EquipmentWorkOrders.js

```diff
@@ -32,20 +33,51 @@ const WO_FILTERS = {
                 return workOrder.mrc && (workOrder.mrc.startsWith("ICF") || workOrder.mrc.startsWith("MTF"));
             })
         }
+    },
+    [WO_FILTER_TYPES.THIS]: {
+        text: WO_FILTER_TYPES.THIS,
+        process: data => [...data]
     }
 }
 
 function EquipmentWorkOrders(props) {
     const { defaultFilter } = props;
 
-    let headers = ['Work Order', 'Description', 'Status', 'Creation Date'];
-    let propCodes = ['number', 'desc', 'status', 'createdDate'];
-    let linksMap = new Map([['number', {linkType: 'fixed', linkValue: 'workorder/', linkPrefix: '/'}]]);
-
-    let [data, setData] = useState([]);
+    let [events, setEvents] = useState([]);
+    let [workorders, setWorkorders] = useState([]);
     let [workOrderFilter, setWorkOrderFilter] = useState(Object.values(WO_FILTER_TYPES).includes(defaultFilter) ? defaultFilter : WO_FILTER_TYPES.ALL)
     const [loadingData, setLoadingData] = useState(true);
 
+    let headers = ['Work Order', 'Equipment', 'Description', 'Status', 'Creation Date'];
+    let propCodes = ['number', 'object','desc', 'status', 'createdDate'];
+
+    if (workOrderFilter === WO_FILTER_TYPES.THIS) {
+        headers = ['Work Order', 'Description', 'Status', 'Creation Date'];
+        propCodes = ['number', 'desc', 'status', 'createdDate'];
+    }
+
+    // make spaces in header strings non-breaking so headers do not wrap to multiple lines
+    headers = headers.map(string => string.replaceAll(' ', '\u00a0'));
```

Comment:
Ideally, this should be achieved with CSS, but I understand that it involves developments in eam-components